src/main/groovy/org/mastercoin/test/TestSupport.groovy

```diff
@@ -265,7 +265,7 @@ trait TestSupport implements MastercoinClientDelegate {
      * @param amount     The number of units to create
      * @return The transaction hash
      */
-    Sha256Hash createProperty(Address address, Ecosystem ecosystem, PropertyType type, Long amount) {
+    Sha256Hash createProperty(Address address, Ecosystem ecosystem, PropertyType type, Number amount) {
```

Comment:
Using `Number` here doesn't seem right to me. Shouldn't we choose the smallest numerical type that will hold the maximum value and then throw an error if that value is exceeded?